engine/api/src/main/scala/pl/touk/nussknacker/engine/api/process/ExpressionConfig.scala

```diff
@@ -17,7 +17,8 @@ case class ExpressionConfig(globalProcessVariables: Map[String, WithCategories[A
                             hideMetaVariable: Boolean = false,
                             strictMethodsChecking: Boolean = true,
                             staticMethodInvocationsChecking: Boolean = false,
-                            disableMethodExecutionForUnknown: Boolean = false
+                            disableMethodExecutionForUnknown: Boolean = false,
+                            disableDynamicPropertyAccess: Boolean = true
                            )
```

Comment:
I wonder, maybe it's time to introduce some "SpelCompilationFlags/Settings" class? To make passing all those flags around easier?

Reply:
Great idea imo. It starts to look trashy. Should we add this class here, or just create another task/branch?

Reply:
I would do that in other change. I have other doubt - using negating logic causes that we are fixing ourselves on opt-out logic. I wonder if it would be better to name it `dynamicPropertyAccessAllowed`. This comment is a little bit late, because I see now that the same is in `disableMethodExecutionForUnknown` (it was first precedent)

Reply:
:+1: let's leave it for next change